apps/mdn/mdn-aws/k8s/celery.workers.deploy.yaml.j2

```diff
@@ -3,11 +3,9 @@
             - /usr/local/bin/newrelic-admin
           args:
             - run-program
-            - ./manage.py
             - celery
+            - "--app=kuma.celery:app"
             - worker
-            - "--events"
-            - "--autoreload"
```

Comment:
Dumping this, since it is deprecated in Celery 4 (and has multiple issues that can be found by doing a Google search of "celery autoreload").